packages/primitives/src/Box/Box.tsx

```diff
@@ -142,24 +80,19 @@ export interface BaseProp {
 export type BoxProps = {
   as?: React.ElementType
 } & BaseProp &
-  VariantProps &
-  SystemProps &
-  IntrinicElements
+  VariantProps
 
 /**
  * Box primitive component which is the base component for
  * all components in Kodiak
  */
-export const Box = styled<'div', BoxProps>('div', {
-  shouldForwardProp,
-})(
+export const Box = styled('div')<BoxProps>(
   {
     boxSizing: 'border-box',
     margin: 0,
     minWidth: 0,
   },
   base,
   variant,
-  ...systemProps,
   sx,
```

Comment:
Should we also forward css as a fallback?  We haven't used the option yet but `theme-ui` passes this as it's last value now.  It could allow us to still use features of emotion.

props => props.css

Reply:
Yea, that's probably not a bad idea to still forward `css`.

Reply:
Chatted in Slack about this, we will wait to add this later, if needed.